### PR TITLE
8300872: WebView's ColorChooser fails to initialize when running in security context

### PR DESCRIPTION
**Pull request by jperedadnr**
This PR removes an unnecessary check that was added as part of [JDK-8267551](https://bugs.openjdk.org/browse/JDK-8267551).

As commented in the [issue](https://bugs.openjdk.org/browse/JDK-8300872), there were no failing/passing tests before/after these lines were added, and there are no actual tests that get into the case.

The mentioned check caused, as a side effect, the JDK-8300872 issue, as running under a (deprecated) security manager fails with:
```
java.security.AccessControlException: access denied ("java.io.FilePermission" "jrt:/javafx.controls/com/sun/javafx/scene/control/skin/modena/pattern-transparent.png" "read")
...
at javafx.graphics/javafx.scene.image.Image.validateUrl(Image.java:1127)
...
```
when `File::exists` is evaluated over an image bundled in a jar.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (2 reviews required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer), 1 [Author](https://openjdk.org/bylaws#author))

### Issue
 * [JDK-8300872](https://bugs.openjdk.org/browse/JDK-8300872): WebView's ColorChooser fails to initialize when running in security context

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Michael Strauß](https://openjdk.org/census#mstrauss) (@mstr2 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx pull/1042/head:pull/1042` \
`$ git checkout pull/1042`

Update a local copy of the PR: \
`$ git checkout pull/1042` \
`$ git pull https://git.openjdk.org/jfx pull/1042/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1042`

View PR using the GUI difftool: \
`$ git pr show -t 1042`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1042.diff">https://git.openjdk.org/jfx/pull/1042.diff</a>

</details>

### CODE REVIEW

**Comment by jperedadnr** [comment]
@kevinrushforth Do you mean a system test case with a custom policy file? I just noticed that `Util::createApplicationLaunchCommand` allows such case.

**Comment by kevinrushforth** [comment]
Ideally, yes. You could add it to the existing `SandboxAppTest` if you like.

**Comment by mstr2** [comment]
> Also, I'd like @mstr2 to comment on the check that you propose to remove.

I’ll get around to doing that next week.

**Comment by jperedadnr** [comment]
I've added a system test, that passes with the proposed fix and fails without it.

Without the fix, the AccessControlException exception prevents the color chooser dialog from showing up, but its window still reports onShowing true and the color can be still changed programmatically. There is an exception logged, but to make the test actually fail, I had to check for the window dimensions.

**Review by kevinrushforth — APPROVED** [review]
Looks good. I left two minor suggestions for changes to the comments in the new policy file.

File: tests/system/src/test/resources/test/sandbox/empty.policy [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Policy file for sandbox tests
+// Use as follows: -Djava.security.policy=test.policy
```
[the remark this answers is not in the document]
Comment (jperedadnr) [reply tests/system/src/test/resources/test/sandbox/empty.policy]: Done

File: tests/system/src/test/resources/test/sandbox/empty.policy [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+// No permissions to read resource files are granted
```
[the remark this answers is not in the document]
Comment (jperedadnr) [reply tests/system/src/test/resources/test/sandbox/empty.policy:4]: Right, I've added only the required permission for the test.

**Review by kevinrushforth — APPROVED** [review]

**Comment by mstr2** [comment]
I think that I added the call to `File.exists` to quickly detect whether a URL refers to a file on disk, but that seems unnecessary in retrospect.

**Review by mstr2 — APPROVED** [review]

**Pull request closed without merging**